keras_cv/layers/preprocessing3d/group_points_by_bounding_boxes.py

```diff
@@ -80,6 +80,13 @@ def __init__(
         self._max_points_per_bounding_boxes = max_points_per_bounding_boxes
         self._auto_vectorize = False
 
+    def get_params(self):
+        return "label_index:{},min_points_per_bounding_boxes:{},max_points_per_bounding_boxes:{}".format(
+            str(self._label_index),
+            str(self._min_points_per_bounding_boxes),
```

Comment:
Actually -- you should probably override `get_config` for layers:
https://keras.io/guides/serialization_and_saving/

Reply:
Done.